src/features/views/components/ViewDataTable/columnTypes/SurveyOptionsColumnType.tsx

```diff
@@ -137,7 +139,7 @@ const Cell: FC<{
         submissions={cell.map((sub, index) => ({
           id: sub.submission_id,
           matchingContent:
-            index == 0
+            index == cell.length - 1
```

Comment:
I don't fully understand this change. Can you explain?

Reply:
This is what a preview of multiple responses looks like on main, the oldest response is the only one that renders the chips: 
![bild](https://github.com/user-attachments/assets/49e97ebf-ec95-4464-a76d-55b1b405ab86)

And this is what it looks like after this fix: 
![bild](https://github.com/user-attachments/assets/45bc0aae-74c0-40fc-a531-dc3bcc3424b6)

The design in Figma looks a bit different, it shows the chips on two of the responses and also has an indicator of overflow ("+5"), but since no such logic existed in the current code I assumed this was for some reason diverged from. I do not completely understand the logic of showing for 2/3 responses, as in the design. 
![bild](https://github.com/user-attachments/assets/02922212-f5c9-4615-be35-f105c27c55be)


My assumption is that what the person writing the code originally was aiming for is to show the chips for the most recent response, but got it backwards.

Reply:
Yeah, I think you're right. Thanks for clarifying!